spec/models/task_spec.rb

```diff
@@ -25,9 +25,10 @@
     end
 
     describe '検索機能' do
-    let!(:task) { FactoryBot.create(:task, title: 'test1', status: "未着手") }
-    let!(:second_task) { FactoryBot.create(:second_task, title: "test2", status: "着手中") }
-    let!(:third_task) { FactoryBot.create(:second_task, title: "sample3", status: "完了") }
+    user = FactoryBot.create(:user, name: 'model_test', email: 'model_test@gmail.com')
```

Comment:
FactoryBotという部分は設定ファイルを変更することで省略することが可能です。
```suggestion
    user = create(:user, name: 'model_test', email: 'model_test@example.com')
```